src/components/Card/Card.js

```diff
@@ -156,31 +147,29 @@ const styles = StyleSheet.create({
     height
   }),
   image: {
-    width: '100%',
-    height: '100%'
+    flex: 1
   },
   content: {
-    justifyContent: 'space-between',
     flex: 1
   },
   title: {
     color: COLORS.black,
     fontWeight: '600',
-    fontSize: 14,
-    lineHeight: 17,
-    marginBottom: 8,
+    fontSize: normalizeFontSizeByWidth(14),
+    lineHeight: dimen.normalizeDimen(17),
+    marginBottom: dimen.normalizeDimen(4),
     fontFamily: fonts.inter[600],
-    paddingRight: 20,
-    paddingLeft: 20
+    paddingRight: dimen.normalizeDimen(20),
+    paddingLeft: dimen.normalizeDimen(20)
   },
   description: {
     color: COLORS.blackgrey,
-    fontSize: 12,
+    fontSize: normalizeFontSizeByWidth(12),
     fontFamily: fonts.inter[400],
-    paddingRight: 20,
-    paddingLeft: 20,
-    marginTop: 5,
-    lineHeight: 18
+    paddingRight: dimen.normalizeDimen(20),
+    paddingLeft: dimen.normalizeDimen(20),
+    marginTop: dimen.normalizeDimen(4),
+    lineHeight: dimen.normalizeDimen(18)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The styles for `image`, `content`, `title`, and `description` have been updated. These changes seem to be aimed at making the layout more flexible and the text more responsive to different screen widths. However, please ensure that these changes meet your design requirements across all target screen sizes.


<!-- This is an auto-generated comment by OSS CodeRabbit -->